modules/distr-tracing-rn-technology-preview.adoc

```diff
@@ -0,0 +1,26 @@
+////
+Module included in the following assemblies:
+* distr-tracing--release-notes.adoc
+////
+
+[id="distr-tracing-rn-tech-preview_{context}"]
+= Technology Preview
+////
+Provide the following info for each issue if possible:
+Description -  Describe the new functionality available to the customer.  For enhancements, try to describe as specifically as possible where the customer will see changes.  Avoid the word “supports” as in [product] now supports [feature] to avoid customer confusion with full support.  Say, for example, “available as a Technology Preview.”
+Package - A brief description of what the customer has to install or enable to use the Technology Preview feature.    (e.g., available in quickstart.zip on customer portal, JDF website, container on registry, enable option, etc.)
+////
+
+[IMPORTANT]
+====
+Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production.
+These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. For more information about the support scope of Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+== {ProductName} 2.0.0 Technology Preview
+
+This release includes the addition of the distributed tracing data collection, which you install using the OpenTelemetry Operator.
+
+{ProductName} data collection is based on OpenTelemetry's Operator and Collector. The Collector can be used to receive traces in either the OpenTelemetry or Jaeger protocol and send the trace data to the OpenShift distributed tracing platform. Other capabilities of the Collector are not supported at this time.
```

Comment:
Possessives for inanimate objects can be tricky. Do you think this reads just as well? 

`...data collection is based on the OpenTelemetry Operator and Collector.`